hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncAdminBuilder.java

```diff
@@ -50,21 +50,38 @@
    * Set the base pause time for retrying. We use an exponential policy to generate sleep time when
    * retrying.
    * @return this for invocation chaining
-   * @see #setRetryPauseForCQTBE(long, TimeUnit)
+   * @see #setRetryPauseForServerOverloaded(long, TimeUnit)
    */
   AsyncAdminBuilder setRetryPause(long timeout, TimeUnit unit);
 
   /**
-   * Set the base pause time for retrying when we hit {@code CallQueueTooBigException}. We use an
+   * Set the base pause time for retrying when we hit {@code ServerOverloadedException}. We use an
```

Comment:
This `ServerLoverloadedException` type doesn't exist anymore, and you make mention to it in several places in the javadocs. Use `{@link _class_}` instead of `@code` and the IDE will support you through class renames and deletions.